solidity/test/bridge/Deployment.test.ts

```diff
@@ -0,0 +1,234 @@
+    })
+
+    it("should set Bridge implementation in ProxyAdmin", async () => {
+      expect(
+        await proxyAdmin.getProxyImplementation(bridgeProxy.address),
+        "invalid proxy implementation"
+      ).to.be.equal(bridgeImplementationAddress)
+    })
+
+    it("should set implementation address different than proxy address", async () => {
+      expect(
+        await bridgeProxy
+          .connect(proxyAdmin.address)
+          .callStatic.implementation(),
+        "invalid implementation"
+      ).to.be.not.equal(bridge.address)
+    })
+
+    it("should set Bridge governance", async () => {
+      expect(await bridge.governance()).to.be.equal(
+        bridgeGovernance.address,
+        "invalid Bridge governance"
+      )
+    })
+
+    it("should set BridgeGovernance owner", async () => {
+      expect(
+        await bridgeGovernance.owner(),
+        "invalid BridgeGovernance owner"
+      ).equal(governance.address)
+    })
+
+    it("should revert when initialize called again", async () => {
+      await expect(
+        bridge.initialize(AddressZero, AddressZero, AddressZero, AddressZero, 0)
+      ).to.be.revertedWith("Initializable: contract is already initialized")
+    })
+
+    it("should set WalletOwner in WalletRegistry", async () => {
+      expect(
+        await walletRegistry.walletOwner(),
+        "invalid WalletOwner in WalletRegistry"
+      ).equal(bridge.address)
+    })
+  })
+
+  describe("Bank", () => {
+    it("should set Bridge reference", async () => {
+      expect(await bank.bridge(), "invalid Bridge address").equal(
+        bridge.address
+      )
+    })
+
+    it("should set Bank owner", async () => {
+      expect(await bank.owner(), "invalid Bank owner").equal(governance.address)
+    })
+  })
+
+  describe("TBTCVault", () => {
+    it("should set Bank reference", async () => {
+      expect(await tbtcVault.bank(), "invalid Bank address").equal(bank.address)
+    })
+
+    it("should set TBTC reference", async () => {
+      expect(await tbtcVault.tbtcToken(), "invalid TBTC address").equal(
+        tbtc.address
+      )
+    })
+
+    it("should set TBTCVault owner", async () => {
+      expect(await tbtcVault.owner(), "invalid TBTCVault owner").equal(
+        governance.address
+      )
+    })
+  })
+
+  describe("MaintainerProxy", () => {
+    it("should set Bridge reference", async () => {
+      expect(await maintainerProxy.bridge(), "invalid Bridge address").equal(
+        bridge.address
+      )
+    })
+
+    it("should set ReimbursementPool reference", async () => {
+      expect(
+        await maintainerProxy.reimbursementPool(),
+        "invalid ReimbursementPool address"
+      ).equal(reimbursementPool.address)
+    })
+
+    it("should set MaintainerProxy owner", async () => {
+      expect(
+        await maintainerProxy.owner(),
+        "invalid MaintainerProxy owner"
+      ).equal(governance.address)
+    })
+  })
+
+  describe("ReimbursementPool", () => {
```

Comment:
We perform some actions on the `VendingMachine` in `03_transfer_vending_machine_roles.ts`.
Shouldn't we also have a separate `describe` for it and test we set `keepTechnicalWalletTeam` as `vendingMachineUpgradeInitiator` and `unmintFeeUpdateInitiator`?

Reply:
This script had been skipped for non-mainnet networks, but I covered it in 0b5d316dc7d8cfb1f0f8867ddac47e844c4c5c19 and 1c75b2c4bbaf3b7bbc3c140e0908fccabb30feca.